### DISCUSSION TITLE
Thank you for your work and the great 'tetra' tutorial

### DISCUSSION THREAD

**Discussion by javaLux** [17cupsofcoffee/tetra#341] [General]
I read the [Rust-Book](https://doc.rust-lang.org/book/title-page.html) and was now looking for a good tutorial for a small Rust project of my own. I stumbled across your 2D game engine and the great tutorial for beginners. It was great to learn and to deepen the basic knowledge about Rust. Thanks for your great work. Keep it up  👨‍🚀 🚀

**Comment by 17cupsofcoffee** [comment]
No problem, glad it helped you!
